src/base/dof_map_constraints.C

```diff
@@ -3856,6 +3856,10 @@ void DofMap::scatter_constraints(MeshBase & mesh)
           ids_offsets[push_i].first = *it;
           ids_offsets[push_i].second = _node_constraints[constrained].second;
         }
+
+      auto & pid_nodes = pushed_nodes[pid];
+      pid_nodes.reserve(nodes.size());
```

Comment:
Shoot.  Didn't I mention that the stuff inside ifdef LIBMESH_ENABLE_NODE_CONSTRAINTS wasn't a good idea for test coverage since it doesn't get hit by CI?  I swear I *intended* to mention that.  Anyway, this is broken, starting with std::set::reserve not being a thing.  I'll see if I can fix.